### ISSUE TITLE
Mixed Precision (FP16, AMP)

### ISSUE THREAD

**Issue by L-Reichardt** [traveller59/spconv#513]
Issue for documentation purposes. Maybe it will help someone else.

I ran a spconv_2.x model with automatic mixed precision, however the tensors in the model went to NAN. A solution was to increase the default eps parameters of BatchNorm and Adam / AdamW to higher F16 compatible values. I haven't found this necessary for other models, but here it worked.

eps BN = 6.2e-5
eps AdamW = 1e-4

Solution in this issue : https://github.com/pytorch/pytorch/issues/40497#issuecomment-1002372380
